pkg/build/api/validation/validation.go

```diff
@@ -123,13 +130,16 @@ func validateBuildSpec(spec *buildapi.BuildSpec, fldPath *field.Path) field.Erro
 	return allErrs
 }
 
-const maxDockerfileLengthBytes = 60 * 1000
+const (
+	maxDockerfileLengthBytes  = 60 * 1000
+	maxJenkinsfileLengthBytes = 60 * 1000
+)
```

Comment:
Yes, we can bump it up for Jenkins probably, but size in etcd is not
unlimited.  I'd be ok up to maybe 100k for jenkinsfile if we think 60k is
too low for most people using this path.

On Fri, Apr 15, 2016 at 9:44 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/api/validation/validation.go
> <https://github.com/openshift/origin/pull/7949#discussion_r59960195>:
>
> > @@ -123,13 +130,16 @@ func validateBuildSpec(spec *buildapi.BuildSpec, fldPath *field.Path) field.Erro
> >  	return allErrs
> >  }
> >
> > -const maxDockerfileLengthBytes = 60 * 1000
> > +const (
> > +	maxDockerfileLengthBytes  = 60 * 1000
> > +	maxJenkinsfileLengthBytes = 60 * 1000
> > +)
>
> 60k should be good enough for anyone?
>
> I think @smarterclayton <https://github.com/smarterclayton> introduced
> the original dockerfile limit.. what was the motivation? concerns about
> etcd storage abuse?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7949/files/15f8dabcad29df00915ad82edd4e0a425894b890#r59960195>
>